src/services/admin/mentor.service.ts

```diff
@@ -75,3 +75,38 @@ export const getAllMentors = async (
     throw new Error('Error getting mentors')
   }
 }
+
+export const findAllMentorEmails = async (
+  status: ApplicationStatus | undefined
+): Promise<{
+  statusCode: number
+  emails?: string[]
+  message: string
+}> => {
+  try {
+    const mentorRepository = dataSource.getRepository(Mentor)
+
+    const allMentors: Mentor[] = await mentorRepository.find({
+      where: status ? { state: status } : {},
+      select: ['profile'],
```

Comment:
Do we have to select the profile here? Doesn't it return the profile from the below relation? Can you check again?

Reply:
fixed thanks